Sources/NIONFS3/NFSTypes+Mount.swift

```diff
@@ -0,0 +1,82 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021-2022 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIOCore
+
+// MARK: - Mount
+public struct NFS3CallMount: Equatable {
```

Comment:
These types all appear to have `Mount` counterparts; are both types used and/or needed?

Reply:
fantastic catch, thank you! removed the NFS ones, they're MOUNT procedures